src/getting-started.md

```diff
@@ -131,204 +125,189 @@ are two major downsides:
 this:
 
     Q: Bonjour
     A: Hello
        Page #12
 
-And:
+And this:
 
     Q: Hello
     A: Bonjour
        Page #12
 
-In this example, we have three pieces of related information: a French
+In this example, we have three related pieces of information: a French
 word, an English meaning, and a page number. If we put them together,
 they’d look like this:
 
     French: Bonjour
     English: Hello
     Page: 12
 
-In Anki, this related information is called a 'note', and each piece of
-information is called a 'field'. So we can say that this type of note
-has three fields: French, English, and Page.
+In Anki, this collection of related information is called a _note_ and each piece of information is contained in a _field_. In this example, the note
+has three fields: "French", "English", and "Page".
 
-To add and edit fields, click the “Fields…​” button while adding or
+To add and edit fields, click the "Fields…​" button while adding or
 editing notes. For more information on fields, please see the
 [Customizing Fields](editing.md#customizing-fields) section.
 
 ### Card Types
 
 In order for Anki to create cards based on our notes, we need to give it
 a blueprint that says which fields should be displayed on the front or
-back of each card. This blueprint is called a 'card type'. Each type of
+back of each card. This blueprint is called a _card type_. Each type of
 note can have one or more card types; when you add a note, Anki will
 create one card for each card type.
 
-Each card type has two 'templates', one for the question and one for the
-answer. In the above French example, we wanted the recognition card to
+All card types have two _templates_, one for the question and one for the
+answer. In the previous French example, we wanted the back of our recognition card to
 look like this:
 
     Q: Bonjour
     A: Hello
        Page #12
 
-To do this, we can set the question and answer templates to:
+To do this, we can set the answer template to:
 
     Q: {{French}}
     A: {{English}}<br>
        Page #{{Page}}
 
-By surrounding a field name in double curly brackets, we tell Anki to
-replace that section with the actual information in the field. Anything
-not surrounded by curly brackets remains the same on each card. (For
-instance, we don’t have to type “Page \#” into the Page field when
-adding material – it’s added automatically to every card.) &lt;br&gt; is
-a special code that tells Anki to move to the next line; more details
-are available in the [templates](templates/intro.md) section.
+In card templates, field names are wrapped in double curly brackets like `{{French}}` or `{{English}}`. Anki replaces those with the actual text the field contain. This is known as ["Field replacement"](/field-replacement.md). Text
+not surrounded by curly brackets appears the same on each card. For
+example, we don't need to enter "Page \#" in every card; it’s added automatically to every card. The `&lt;br&gt;` tag is
```

Comment:
I suggest we remove the word field and the semicolon to simply say "we don't need to enter "Page \#" on every note as it's added automatically to every card."

 It looks a bit weird to have "every note" and "every card" together but I believe this is better.

```suggestion
In card templates, field names are wrapped in double curly brackets, like `{{French}}` or `{{English}}`. Anki replaces those with the actual text the fields contain. This is called a ["Field replacement"](templates/fields.md). Text not wrapped in double curly brackets appears the same on each card. For example, we don't need to enter "Page \#" on every note as it’s added automatically to every card. The `&lt;br&gt;` tag is
```

Reply:
How about this? 
```suggestion
In card templates, field names are wrapped in double curly brackets, like `{{French}}` or `{{English}}`. Anki replaces those with the actual text the fields contain. This is called a ["Field replacement"](templates/fields.md). Text not wrapped in double curly brackets appears the same on each card. For example, we won't need to add "Page \#" on every note because the template will add it automatically to every card. The `&lt;br&gt;` tag is
```